### PR TITLE
Feature - Address in menu

### PR DESCRIPTION
**Pull request by MaxStalker**
Implements #5 
Additionaly adds address to menu.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -859,203 +859,203 @@ export default class App extends Component {
...
+                    this.setState({
+                      parsingTheChain: false,
+                      loadedBlocksTop: upperBoundOfSearch
                     });
-                  }
-                });
-              }
-            }}
-          />
+                    //console.log("~~ DONE PARSING SET ~~")
```
Comment (johba37) [inline src/App.js]: remove code comments

File: src/App.js [hunk 2]
```diff
@@ -859,203 +859,203 @@ export default class App extends Component {
...
-                        );
+                      console.log(
+                        " +++++++======== Parsing from " +
+                          loadedBlocksTop +
+                          " to " +
+                          upperBoundOfSearch +
+                          "...."
+                      );
+                      while (loadedBlocksTop < parseBlock) {
+                        //console.log(" ++ Parsing Block "+parseBlock+" for transactions...")
```
Comment (johba37) [inline src/App.js]: no code comments

**Review by johba37 — CHANGES_REQUESTED** [review]
can't copy the address on mobile or desktop

**Comment by MaxStalker** [comment]
> can't copy the address on mobile or desktop

Apparently selection background was disabled. I fixed that one.

**Review by johba37 — APPROVED** [review]

**Pull request merged**